sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/_transport.py

```diff
@@ -384,12 +384,12 @@ def close(self):
             except Exception as exc:
                 # TODO: shutdown could raise OSError, Transport endpoint is not connected if the endpoint is already
```

Comment:
checking that we're intentionally leaving the TODO here?

Reply:
Yes intentionally leaving this one as that particular OSError occurs very often and in code using that flow they just catch the exception and pass.